### DISCUSSION TITLE
A way to tell *why* backups are unhealthy

### DISCUSSION THREAD

**Discussion by hailwood** [spatie/laravel-backup#1509] [Ideas]
Running `php artisan backup:monitor` with my backups totally over 5GB (my configured limit) I get the rather unhelpful message of
> The backups on db-backups are considered unhealthy!

It would be great if we could something to tell us which check failed. E.g.
> The backups on db-backups are considered unhealthy:
> - Total size > 5GB

Or even as a bare minimum 
> The backups on db-backups are considered unhealthy (Health check: MaximumStorageInMegabytes)

**Comment by freekmurze** [comment]
Great idea, feel free to PR this! Be sure to include tests.
